README.md

```diff
@@ -1,46 +1,160 @@
+| `sympa_log_level`                          | `0`                                                                                                                   | Log verbosity  0: normal, 2,3,4: for debug                                                                                                                                                                                                                                                                                               |
+| `sympa_log_socket_type`                    | `unix`                                                                                                                | Communication mode with syslogd (unix,inet)                                                                                                                                                                                                                                                                                              |
+| `sympa_sendmail`                           | `/usr/bin/sendmail`                                                                                                   | Path to the MTA (sendmail, postfix, exim or qmail)  should point to a sendmail-compatible binary (eg: a binary named "sendmail"  is distributed with Postfix)                                                                                                                                                                            |
+| `sympa_maxsmtp`                            | `40`                                                                                                                  | Max. number of Sendmail processes (launched by Sympa) running                                                                                                                                                                                                                                                                            |
+| `sympa_log_smtp`                           | `"off"`                                                                                                               |                                                                                                                                                                                                                                                                                                                                          |
+| `sympa_use_blacklist`                      | `[ send, create_list ]`                                                                                               | list of operations for which blacklist filter is applied  Setting this parameter to "none" will hide the blacklist feature                                                                                                                                                                                                               |
```

Comment:
`none` is not possible with the role. I would remove the sentence and set the value in jinja if the list is empty.